contrib/babelfishpg_common/sql/rowversion.sql

```diff
@@ -172,7 +172,7 @@ LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
 CREATE CAST (sys.ROWVERSION as INT8)
 WITH FUNCTION sys.rowversionint8 (sys.ROWVERSION) AS ASSIGNMENT;
 
-CREATE DOMAIN sys.TIMESTAMP AS sys.ROWVERSION;
```

Comment:
Why havent we added NOT NULL to sys.ROWVERSION itself?

AFAIK even rowversion should be not null, how are we ensuring that is fixed?

Reply:
`A nonnullable rowversion column is semantically equivalent to a binary(8) column. A nullable rowversion column is semantically equivalent to a varbinary(8) column.` 
Got this from sql server documentation .
From this we can see that rowversion can be null.